pkg/pillar/cmd/baseosmgr/baseosmgr.go

```diff
@@ -506,11 +506,8 @@ func initializeGlobalConfigHandles(ctx *baseOsMgrContext) {
 	if err != nil {
 		log.Fatal(err)
 	}
-	subGlobalConfig.MaxProcessTimeWarn = warningTime
-	subGlobalConfig.MaxProcessTimeError = errorTime
-	subGlobalConfig.ModifyHandler = handleGlobalConfigModify
-	subGlobalConfig.CreateHandler = handleGlobalConfigModify
-	subGlobalConfig.DeleteHandler = handleGlobalConfigDelete
+	subGlobalConfig.Handlers(handleGlobalConfigModify, handleGlobalConfigModify, handleGlobalConfigDelete, nil, nil)
+	subGlobalConfig.TimeLimits(warningTime, errorTime)
```

Comment:
Another comment since you are anyway touching this..

subGlobalConfig.TimeLimits(warningTime, errorTime)
  --> Only a couple of agents change these time limits. The rest just use a default value. So we should follow that model.. Where if an agent just need default values, it doesn't need to set these..

Reply:
I thought about this one too. I wanted to have a default at the implementation level, that way it can just override. These already were doing 

```
sub.MaxProcessErrorTime = something
sub.MaxProcessWarningTime = somethingElse
```

so I kept it minimal. If we adopt your approach from the previous comment, this rolls right in.